### DISCUSSION TITLE
Need shipping confirmation for customs clearance

### DISCUSSION THREAD

**Discussion by hypnoglow** [aquasecurity/community#42] [Q&A]
Hi,

I think I won a prize for participating in Aqua Security Hacktoberfest (got the confirmation on my email).

Recently I received a notification from UPS Delivery Service. They requested documents regarding the shipping required for customs clearance in Russia.

Is it possible to get any documents from you confirming this parcel? Or at least could you send me an email which would contain confirmation that the parcel is a gift I won and my shipping details? Delivery service operator whom I called said that this may be enough if no other documents can be provided.

Thanks!

**Comment by itaysk** [comment]
Hi @hypnoglow ,
Just to make sure, is UPS asking you for documents, or UPS is asked for documents by your country customs? We have asked UPS to take a closer look at your shipment, hope they can resolve this issue quickly. I don't have a shipping certificate to send to you, but if it helps I can send you a simple email confirming that we have sent you a package.

**Reply by hypnoglow** [reply]
> Just to make sure, is UPS asking you for documents, or UPS is asked for documents by your country customs?

As I understand, customs holds the shipping until they receive confirmation, and UPS is asking me for documents, because they don't have any. When I called delivery service operator, they said that this is a standard procedure for international shipping, probably related to taxes or duty-free thresholds in Russia. But it is the first time I'm facing this, probably because I have never received packages via UPS.

> I don't have a shipping certificate to send to you, but if it helps I can send you a simple email confirming that we have sent you a package.

This would be great. I hope that this will be enough to push the shipping forward.

**Reply by itaysk** [reply]
Actually, all eligible participants were supposed to get an email on Jan 4th saying that you should expect the package. did you get that?

**Reply by hypnoglow** [reply]
I got that email, but it has neither personal details like my name or shipping address nor package description, so I think this letter would not be an appropriate proof.

**Reply by itaysk** [reply]
ok. I'll send you an email with those details as well

**Reply by hypnoglow** [reply]
Got it, thank you very much! I'm going to submit it today. I'll let you know if it works. 🙏

**Reply by ashish2675** [reply]
I have been asked to pay a custom duty of ₹500 for the Hactoberfest swags whereas my friends got it for free in India why is it so?
